src/components/molecules/Card/stories/card.stories.tsx

```diff
@@ -223,6 +224,10 @@ export const CustomTitles = () => {
             type='image'
             imageUrl='https://llandscapes-10674.kxcdn.com/wp-content/uploads/2019/07/lighting.jpg'
             linkUrl='https://youtube.com'
+            tag='tag'
+            tagStyle={{ backgroundColor: 'var(--color-primary)' }}
+            contentTypeIconStyle={{ backgroundColor: 'var(--color-primary)' }}
```

Comment:
Would be preferable to avoid inline-styles and wrap the `Card` in a styled-component and use the `.tag-button` and `.content-type-icon` class.